tests/FSharp.Compiler.ComponentTests/Interop/StaticsInInterfaces.fs

```diff
@@ -606,6 +606,36 @@ module Test =
 #endif
         ]
 
+    [<FactForNETCOREAPP>]
+    let ``F# can call overwritten static virtual member from interface``() =
+        let CSharpLib =
+            CSharp """
+namespace Test;
+
+public interface I
+{
+    static virtual string Echo(string x) => x;
+}
+            """
+            |> withCSharpLanguageVersion CSharpLanguageVersion.CSharp11
+            |> withName "CsLibAssembly"
+
+        FSharp """
+type Imp() =
+    interface Test.I with
+        static member Echo (x: string) = x + "_imp"
+
+let echo<'T when 'T :> Test.I> x = 'T.Echo(x)
+
+if echo<Imp> "a" <> "a_imp" then
```

Comment:
On order to make what this does a tiny bit clearer, I would modify this test case slightly:
````

type Imp() =
    interface Test.I with
        static member Echo (x: string) = x + "_imp"

let echo<'T when 'T :> Test.I> x = 'T.Echo(x)
let result = echo<Imp> "a"

if result = "a_imp" then
    printf $"Succeeded"
elif result = "a" then
    failwith "Incorrectly invoked the base class 'Echo'"
else
    failwith "incorrect value"
````

Reply:
Updated in b16cf8351